index.js

```diff
@@ -1,4 +1,33 @@
-class Stack {}
+class Stack {
+  #list = [];
+
+  constructor(initialList) {
+    if (initialList) this.#list = initialList;
+  }
+
+  // methods
+
+  pushItem(item) {
+    this.#list.push(item);
+  }
+
+  removeLast(item) {
+    this.#list.pop(item);
+  }
+
+  topItem(item) {
+    this.#list.peek(item);
+    //this.#list[this.#list.length - 1]
+  }
```

Comment:
I also think you probably realize this by now but `.peek()` isn't an array method but i'm glad to see y'all added the way Julie solved it(maybe even before she showed us how!).